### PR TITLE
Svg color change

### LINKED ISSUE #747: 🐛 [Bug report]: Making the language logo colorfull on hover
### Descrcibe the bug?

Currently when we hover on the trending play the language logo is not colored to be more visible.

### Steps to reproduce the bug?

Go to the home page of reactplay.io scroll down to the treading plays section and then hover on any one of the play you will notice that the play card's size increased but the language icon remains the same.

### Expected behavior

When we hover on the play card the language icon should become colorful.

### Desktop (Please provide your system information)

- Windows
- Chrome

### Mobile (Please provide your device information)

_No response_

### Screenshot / Screenshare

image

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Hat52**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description
The color of the language logo inside the card now changes on the hover of the card. 

Fixes #747 

## Type of change

Please delete options that are not relevant.

- [ x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- locally.

# Checklist:

- [ x] I have performed a self-review of my own code
- [ x] My changes generate no new warnings

### CODE REVIEW

File: src/common/playlists/playlist.css [hunk 1]
```diff
@@ -62,9 +62,22 @@
   opacity: 0.2;
 }
 
+.list-plays li a:hover  .play-header .language {
+  opacity:0.8;
+}
+
+.list-plays li a:hover .play-header .language-ts{
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='20' data-name='Layer 1' fill='blue' viewBox='6 6 36 36'%3E%3Cpath d='M6,6V42H42V6ZM26.9,25H22.21V38.76h-3V25H14.57V22.46H26.9ZM32.29,39a9.42,9.42,0,0,1-4.41-1l.66-2.46a8.29,8.29,0,0,0,3.94,1c1.9,0,3-.89,3-2.22s-.82-2-2.89-2.71c-2.69-.95-4.41-2.39-4.41-4.73,0-2.67,2.23-4.68,5.77-4.68a8.66,8.66,0,0,1,3.91.81l-.71,2.4a7,7,0,0,0-3.26-.78c-1.89,0-2.69,1-2.69,2,0,1.25.94,1.84,3.11,2.67,2.84,1.08,4.2,2.5,4.2,4.84C38.47,36.74,36.49,39,32.29,39Z'/%3E%3C/svg%3E");
+}
```
Comment (Sachin-chaurasiya) [inline src/common/playlists/playlist.css:71]:
```suggestion
  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='20' data-name='Layer 1' fill='#2F73BF' viewBox='6 6 36 36'%3E%3Cpath d='M6,6V42H42V6ZM26.9,25H22.21V38.76h-3V25H14.57V22.46H26.9ZM32.29,39a9.42,9.42,0,0,1-4.41-1l.66-2.46a8.29,8.29,0,0,0,3.94,1c1.9,0,3-.89,3-2.22s-.82-2-2.89-2.71c-2.69-.95-4.41-2.39-4.41-4.73,0-2.67,2.23-4.68,5.77-4.68a8.66,8.66,0,0,1,3.91.81l-.71,2.4a7,7,0,0,0-3.26-.78c-1.89,0-2.69,1-2.69,2,0,1.25.94,1.84,3.11,2.67,2.84,1.08,4.2,2.5,4.2,4.84C38.47,36.74,36.49,39,32.29,39Z'/%3E%3C/svg%3E");
}
```

File: src/common/playlists/playlist.css [hunk 2]
```diff
@@ -62,9 +62,22 @@
...
 }
+
+.list-plays li a:hover .play-header .language-js{
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='20' viewBox='0 0 20 20'%3E%3Cpath fill='blue' fill-rule='evenodd' d='M379.328 7337.432c-1.745 0-2.873-.832-3.423-1.92l1.53-.886c.403.658.926 1.141 1.853 1.141.778 0 1.275-.389 1.275-.926 0-.808-1.078-1.124-1.839-1.45-1.356-.577-2.256-1.302-2.256-2.833 0-1.409 1.074-2.483 2.753-2.483 1.194 0 2.054.416 2.671 1.503l-1.463.94c-.322-.577-.671-.805-1.208-.805-.551 0-.9.349-.9.805 0 .564.349.792 1.155 1.141 1.689.724 2.967 1.293 2.967 3.155 0 1.692-1.329 2.618-3.115 2.618zm-4.328-2.833c0 1.947-1.199 2.976-2.864 2.976-1.504 0-2.405-.575-2.848-1.575h-.026l1.529-1.069c.295.523.561.894 1.205.894.618 0 1.004-.313 1.004-1.252V7328h2v6.599zM364 7339h20v-20h-20v20z' transform='translate(-364 -7319)'/%3E%3C/svg%3E");
+}
```
Comment (Sachin-chaurasiya) [inline src/common/playlists/playlist.css:79]:
```suggestion
.list-plays li a:hover .play-header .language-js{
  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='20' viewBox='0 0 20 20'%3E%3Cpath fill='#EFD81D' fill-rule='evenodd' d='M379.328 7337.432c-1.745 0-2.873-.832-3.423-1.92l1.53-.886c.403.658.926 1.141 1.853 1.141.778 0 1.275-.389 1.275-.926 0-.808-1.078-1.124-1.839-1.45-1.356-.577-2.256-1.302-2.256-2.833 0-1.409 1.074-2.483 2.753-2.483 1.194 0 2.054.416 2.671 1.503l-1.463.94c-.322-.577-.671-.805-1.208-.805-.551 0-.9.349-.9.805 0 .564.349.792 1.155 1.141 1.689.724 2.967 1.293 2.967 3.155 0 1.692-1.329 2.618-3.115 2.618zm-4.328-2.833c0 1.947-1.199 2.976-2.864 2.976-1.504 0-2.405-.575-2.848-1.575h-.026l1.529-1.069c.295.523.561.894 1.205.894.618 0 1.004-.313 1.004-1.252V7328h2v6.599zM364 7339h20v-20h-20v20z' transform='translate(-364 -7319)'/%3E%3C/svg%3E");
}
```

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]

**Comment by Hat52** [comment]
@Sachin-chaurasiya I have update the value of blue to its hex value. As the # is not supported in fill so I had to use %23 instead of #.

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya I have update the value of blue to its hex value. As the # is not supported in fill so I had to use %23 instead of #.
> 

@Hat52 , Typescript and JavaScript colour would be different so use the same hex code that I have suggested.

**Comment by Hat52** [comment]
@Sachin-chaurasiya Changed the color of ts.

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya Changed the color of ts.

@Hat52 , You have added the wrong colour codes, please add the below codes.

- Typescript - #2F73BF
- Javascript- #EFD81D

File: src/common/playlists/playlist.css (hunk 1 above)
Comment (Sachin-chaurasiya) [reply src/common/playlists/playlist.css]: @Hat52 , For Typescript.

**Comment by Hat52** [comment]
@Sachin-chaurasiya Done

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by atapas — APPROVED** [review]

**Pull request merged**